_episodes/01-format-data.md

```diff
@@ -107,24 +107,24 @@ So, instead we should have:
 ![single-info example](../fig/single-info.png)
 
 > ## Exercise
-> We're going to take a messy version of the survey data and clean it up.
+> We're going to take a messy version of the survey data and describle how we would clean it up.
 >
 > - Download the data by clicking [here](https://ndownloader.figshare.com/files/2252083) to get it from FigShare.
-> - Open up the data in a spreadsheet program 
+> - Open up the data in a spreadsheet program. 
 > - You can see that there are two tabs. Two field assistants conducted the surveys, one
 in 2013 and one in 2014, and they both kept track of the data in their own way. Now
 you're the person in charge of this project and you want to be able to 
 start analyzing the data.   
-> - With the person next to you, work on the messy data so that a computer will
-be able to understand it. Clean up the 2013 and 2014 tabs, and put them all together
+> - With the person next to you, discuss how to organize the messy data so that a computer will
+be able to understand it. Explain the steps you would take to clean up the 2013 and 2014 tabs, and to put them all together
 in one spreadsheet. 
```

Comment:
Identify what is wrong with this spreadsheet. Also discuss the steps you would need to take to clean up...